Header header
string parent_frame_id
string object_frame_id
float32 bb_dx
float32 bb_dy
float32 bb_dz
duration time_since_start
bool object_lost

